aspnetcore/web-api/advanced/api-analyzer.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using web API analyzers
+author: pranavkm
+description: Learn about analyzers in Microsoft.AspNetCore.Mvc.Api.Analyzers
+ms.author: pranavkm
```

Comment:
Add the following metadata above this line, so that the doc is only available for ASP.NET Core 2.2+:

```
monikerRange: '>= aspnetcore-2.2'
```